test/configuration/configuration.test.ts

```diff
@@ -37,6 +39,27 @@ suite('Configuration', () => {
     assert.deepStrictEqual(normalizedKeybinds[0].after, ['o', '<Esc>', 'k']);
   });
 
+  test.only('textwidth is configurable per-language', async () => {
```

Comment:
```suggestion
  test('textwidth is configurable per-language', async () => {
```

Reply:
Ah whoops, didn't mean to let that make it into the commit. 😅

Reply:
I've probably put the same on `master` a dozen times by now 🙃